### DISCUSSION TITLE
can draw the special symbol?

### DISCUSSION THREAD

**Discussion by Coloryr** [SixLabors/ImageSharp.Drawing#182] [Q&A]
draw in the image,like ❤

**Comment by JimBobSquarePants** [comment]
Yeah that’s text. Use `DrawText`.

**Reply by Coloryr** [reply]
![6~RZJ%A7YVZMW5M 9 3R9NU](https://user-images.githubusercontent.com/26276037/144967240-10b992ff-4841-4c24-9ba6-53a6c91e5e4c.png)
it is font problem?

**Reply by Coloryr** [reply]
my test font is `微软雅黑`

**Reply by JimBobSquarePants** [reply]
I don't speak Chinese but I'm assuming you mean. Microsoft YaHei. That font does not contain emojis. You need to assign a fallback font that contains emojis like Segoe UI Emoji. I also strongly recommend using at least `1.0.0-beta13.16` from MyGet.

**Reply by Coloryr** [reply]
can use tow font to draw?
like use `Microsoft YaHei` as main font,and `Segoe UI Emoji` as secondary font.

**Reply by JimBobSquarePants** [reply]
Yes. Use the `TextOptions.FallbackFontFamilies` property in the version I mentioned above. You can add multiple fallback fonts.

**Reply by Coloryr** [reply]
it work, but it have some error
![1S4 L{K2Z~4$4YXL$WI00KS](https://user-images.githubusercontent.com/26276037/145150461-d43d1094-bfd7-47a1-83c4-2c1abda1bec4.png)
.

**Reply by JimBobSquarePants** [reply]
Without a code sample I have absolutely no ability to help you.

**Reply by Coloryr** [reply]
in [here](https://github.com/Coloryr/McPing/blob/5c75ffbbd6cfeb3b0dba6ce24eb3a74dda73362f/McPing/GenShow.cs#L193)

**Reply by Coloryr** [reply]
I found problem.
![$AUCFI)1PL@ JNA~BP`_52Q](https://user-images.githubusercontent.com/26276037/145154548-c7adf9a2-9b9d-4c5c-b454-5c9c3fde2ba1.png)

`TextMeasurer.Measure`can't calculate the symbol size.

**Reply by JimBobSquarePants** [reply]
You’re not using the version I asked you to use.
